ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -20,7 +20,7 @@ public void WhenTwoLocationsInstantiated_BeEqual(int x1, int y1, int x2, int y2)
 
         [Theory]
         [InlineData(0, 0, 1, 0)]
-        public void WhenTwoLocationsInstantiated_NotBeEqual(int x1, int y1, int x2, int y2)
+        public void GivenTwoNonIdenticalLocations_NotBeEqual(int x1, int y1, int x2, int y2)

```

Comment:
name suggestion, more cases

```suggestion
        [InlineData(1, 2, 3, 4)]
        public void NotBeEqual_GivenLocationsOfDifferingCoordinates(int x1, int y1, int x2, int y2)
```

Reply:
won't give similar name suggestion again, can follow the pattern @indianbread suggested i.e. `ClassShould_BeOrDoThis_GivenCondition`